### PR TITLE
System CAs and Certificate tooltip updates

### PR DESCRIPTION
**Pull request by miker**
(CAs and Certificates use the same tooltips)
Tooltips Updated
Signing Certificate Authority
Key Length
Digest Algorithm
Lifetime
Country
Locality
Common Name
Subject Alternate Names
Certificate
Private Key

webui builds without issue

https://redmine.ixsystems.com/issues/37606

### CODE REVIEW

File: src/app/pages/system/ca/ca-add/ca-add.component.ts [hunk 1]
```diff
@@ -127,7 +129,8 @@ export class CertificateAuthorityAddComponent {
       type : 'input',
       name : 'city',
       placeholder : T('Locality'),
-      tooltip: T('Enter the location of the organization.'),
+      tooltip: T('Enter the location of the organization. For example\
```
Comment (Mrt134) [inline src/app/pages/system/ca/ca-add/ca-add.component.ts]: > For example\

Missing a comma?

File: src/app/pages/system/ca/ca-add/ca-add.component.ts [hunk 2]
```diff
@@ -167,15 +170,17 @@ export class CertificateAuthorityAddComponent {
       type : 'textarea',
       name : 'san',
       placeholder: T('Subject Alternate Names'),
-      tooltip: T('Multi-domain support. Enter additional space separated\
-                  domains.'),
+      tooltip: T('Multi-domain support. Enter additional domains to\
+                  secure, separated by spaces. For example if the\
```
Comment (Mrt134) [inline src/app/pages/system/ca/ca-add/ca-add.component.ts]: >  For example if the\

Missing comma

File: src/app/pages/system/ca/ca-add/ca-add.component.ts (hunk 1 above)
Comment (miker) [reply src/app/pages/system/ca/ca-add/ca-add.component.ts]: added

File: src/app/pages/system/ca/ca-add/ca-add.component.ts (hunk 2 above)
Comment (miker) [reply src/app/pages/system/ca/ca-add/ca-add.component.ts]: added

**Pull request merged**